src/seedu/addressbook/Main.java

```diff
@@ -81,7 +81,7 @@ private void runCommandLoopUntilExitCommand() {
         Command command;
         do {
             String userCommandText = ui.getUserCommand();
-            command = new Parser().parseCommand(userCommandText);
+            command = Parser.parseCommand(userCommandText);
```

Comment:
Yes, good job here. We should just calling the class method without the need to create an instance.